internal/constants/User.go

```diff
@@ -0,0 +1,11 @@
+package constants
+
+const (
+	AdminUser  string = "admin"
+	SystemUser string = "system"
+)
+
+const (
+	AdminUserId  = 2 // we have established Admin user as 2 while setting up devtron
+	SystemUserId = 1 // we have established System user as 1 while setting up devtron, which are being used for auto-trigger operations
```

Comment:
can shift it to userBean, we do not follow this convention in general

Reply:
DONE